meshtastic/mesh.proto

```diff
@@ -1098,13 +1098,13 @@ message MeshPacket {
   /*
    * Describes whether this packet passed via MQTT somewhere along the path it currently took.
    */
-  bool via_mqtt = 14;
+  optional bool via_mqtt = 14;
 
   /*
    * Hop limit with which the original packet started. Sent via LoRa using three bits in the unencrypted header.
    * When receiving a packet, the difference between hop_start and hop_limit gives how many hops it traveled.
    */
-  uint32 hop_start = 15;
+  optional uint32 hop_start = 15;
```

Comment:
Not sure either of these is relevant since they come right out of the mesh header that can't distinguish 0 from unset for sure. It's possible for hop_start, at least, that the firmware could be made to compare it and hop_limit -- if hop_start is 0 and hop_limit isn't, it's clearly not a real value.